sentry-types/src/protocol/v7.rs

```diff
@@ -1368,6 +1368,12 @@ impl fmt::Display for SpanId {
     }
 }
 
+impl fmt::Debug for SpanId {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        write!(fmt, "{}", self)
+    }
+}
```

Comment:
The Debug should probably indicate the type of this value. For example:


```suggestion
impl fmt::Debug for SpanId {
    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
        write!(fmt, "SpanId({})", self)
    }
}
```

Also, minor nit: as far as I can tell, this change appears to be unrelated to the rest of the changes in this PR, so likely it would have been better to split it off as a separate PR. I think it is okay to keep it here this time, but for the future, it is best to try to keep PRs limited in scope to only one change.